ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestExecutorUtil.java

```diff
@@ -342,6 +348,24 @@ private static ExecuteResultAndSortedSources execute(
         Optional.empty());
   }
 
+  @SuppressWarnings("unchecked")
+  private static ExecuteResult executeConfiguredStatement(
+      final KsqlExecutionContext executionContext,
+      final ConfiguredStatement<?> stmt) {
+    final KsqlPlan plan = executionContext.plan(executionContext.getServiceContext(), stmt);
+    final ConfiguredKsqlPlan configuredPlan;
+    try {
+      final String serialized = PLAN_MAPPER.writeValueAsString(plan);
+      configuredPlan = ConfiguredKsqlPlan.of(
+          PLAN_MAPPER.readValue(serialized, KsqlPlan.class),
+          stmt.getOverrides(),
+          stmt.getConfig());
+    } catch (final IOException e) {
+      throw new RuntimeException(e);
+    }
+    return executionContext.execute(executionContext.getServiceContext(), configuredPlan);
+  }
+
```

Comment:
It's a style thing, but I generally think that if you've got some part of the code in a function that wants its own try/catch, it generally makes the code more readable to move it to its own function, e.g. 

```java
  private static ExecuteResult executeConfiguredStatement(
      final KsqlExecutionContext executionContext,
      final ConfiguredStatement<?> stmt
  ) {
    final KsqlPlan plan = executionContext.plan(executionContext.getServiceContext(), stmt);
    final ConfiguredKsqlPlan configuredPlan = buildConfiguredKsqlPlan(stmt, plan);
    return executionContext.execute(executionContext.getServiceContext(), configuredPlan);
  }

  private static ConfiguredKsqlPlan buildConfiguredKsqlPlan(
      final ConfiguredStatement<?> stmt,
      final KsqlPlan plan
  ) {
    try {
      final String serialized = PLAN_MAPPER.writeValueAsString(plan);
      return ConfiguredKsqlPlan.of(
          PLAN_MAPPER.readValue(serialized, KsqlPlan.class),
          stmt.getOverrides(),
          stmt.getConfig()
      );
    } catch (final IOException e) {
      throw new RuntimeException(e);
    }
  }
```